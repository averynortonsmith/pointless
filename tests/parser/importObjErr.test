
x = {
  x = 123
  import "asdf" as a
}