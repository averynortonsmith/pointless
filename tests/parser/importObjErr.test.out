-------------------------------------------------------------------------------
Parser Error:

Expected 'Name', got 'Import'
-------------------------------------------------------------------------------

At (line 4 column 3) in 'tests/parser/importObjErr.test'
  import "asdf" as a
  ^
